server/backend/src/cameras/cameras.service.ts

```diff
@@ -1,27 +1,79 @@
-import { Injectable } from '@nestjs/common';
-import { EntityRepository } from '@mikro-orm/core';
+import { Injectable, Module } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
+import { Collection, EntityRepository } from '@mikro-orm/core';
 import { InjectRepository } from '@mikro-orm/nestjs';
+import { NotFoundError } from '../errors.types';
 import { Notification } from './notification.entity';
 import { Camera } from './camera.entity';
-import { NotFoundError } from '../errors.types';
+import { Expo } from 'expo-server-sdk';
+import * as dotenv from 'dotenv';
+dotenv.config();
 
 @Injectable()
 export class CamerasService {
   constructor(
-    @InjectRepository(Notification)
-    private notificationRepository: EntityRepository<Notification>,
     @InjectRepository(Camera)
     private cameraRepository: EntityRepository<Camera>,
+    @InjectRepository(Notification)
+    private notificationRepository: EntityRepository<Notification>,
   ) {}
 
-  sendNotification(id: string): boolean {
-    /* TODO: verify that ID refers to a valid camera and add a notification to DB */
-    return false;
+  /**
+   * Sends a notification to the user's phone and adds it to the
+   * collection of notifications the user has.
+   * @param idCam
+   */
+  public async sendNotification(idCam: string): Promise<boolean> {
```

Comment:
nit: We only care that the function runs successfully, and we don't return false anywhere, so we should use a `void` promise. This would require some test changes though, so I'm okay with leaving it for now and refactoring later.
```suggestion
  public async sendNotification(idCam: string): Promise<void> {
```

Reply:
Discussed offline, not worth the effort